InfrastructureManager/system_manager.py

```diff
@@ -102,7 +103,8 @@ def get_memory_usage(self, secret):
     mem_stats_dict = { JSONTags.MEMORY :
       {
         JSONTags.AVAILABLE : mem_stats.available,
-        JSONTags.USED : mem_stats.used
+        JSONTags.USED : mem_stats.used,
+        JSONTags.TOTAL : mem_stats.total
```

Comment:
Please remove the changes in the SystemManager. They should be part of the shift in using `get_all_stats` in AppController.